assets/js/src/form_editor/store/selectors.jsx

```diff
@@ -17,6 +17,12 @@ export default {
   getFormSettings(state) {
     return state.formData.settings;
   },
+  placeFormBellowAllPages(state) {
```

Comment:
So far we have been using `getSomething` naming convention for selectors. Why did you decide to change it? Shouldn't we change it for all?

Reply:
I didn't decide to change it. I didn't know about the convention. It makes sense for most of the fields. I thought if it returns boolean it makes sense to omit the `get` prefix. You think all of those functions should be prefixed with `get`?

Reply:
We probably don't have to prefix all. I think that it would be good so that these methods were named as getters. Name `placeFormBellowAllPages` doesn't sound to me as a getter name but it might be just my problem. You decide.